README.md

```diff
@@ -304,13 +304,23 @@ On self-hosted runners, you also might want to alter the default pixi install lo
     pixi-bin-path: ${{ runner.temp }}/bin/pixi
 ```
 
+### Using the `pyproject.toml` as a manifest file for pixi.
+`setup-pixi` wil automatically pick up the `pyproject.toml` if it contains a `[tool.pixi.project]` section and no `pixi.toml`.
```

Comment:
```suggestion

`setup-pixi` will automatically pick up the `pyproject.toml` if it contains a `[tool.pixi.project]` section and no `pixi.toml`.
```

Reply:
Do you want to keep the check?

Reply:
Yeah I think it doesn't hurt.

Reply:
I was unaware of the whole dist logic. This makes it much easier :smile: